tests/ut/backends/test_memory.py

```diff
@@ -187,8 +187,8 @@ async def test_clear_no_namespace(self, memory):
         SimpleMemoryBackend._handlers = "asdad"
         SimpleMemoryBackend._cache = "asdad"
         await memory._clear()
-        SimpleMemoryBackend._handlers = {}
-        SimpleMemoryBackend._cache = {}
+        assert {} == SimpleMemoryBackend._handlers
+        assert {} == SimpleMemoryBackend._cache
```

Comment:
I like to follow the make sure X is equals to Y where Y is what you expect and X is the variable, keep it this way please :)

Reply:
ah wow... haha good one